1xx-enable-ssl-for-kafka-bridge.md

```diff
@@ -0,0 +1,148 @@
+
+In the future, we are also likely to implement authentication and authorization for HTTP Bridge. By having separate listeners, the internal endpoints will not be affected by such changes and will make it easier to implement them.
+
+### Automating certificate management
+
+Users can choose to allow the operator to manage the certificate instead of providing their own certificate and key. This can be implemented by adding another boolean field on the KafkaBridge CR such as `generateCertificate`. The operator then could issue the certificate using the same CA for their Kafka cluster as we do today for internal components such as Cruise Control and Kafka Exporter, however with the upcoming support for cert-manager, the way the operator manages certificates will change. This is the reason we are leaving this feature for future improvement.
+
+## Affected/not affected projects
+
+- strimzi-kafka-bridge
+- strimzi-cluster-operator
+
+## Compatibility
+
+This will introduce a breaking change for users who collect metrics on port 8080 or the port defined by `http.port` configuration. Also users who run Bridge on baremetals and do their own manual healtchecks using `/healthy` and `/ready` endpoints will be affected as well. These users have to reconfigure their healthcheck and monitoring system to use port 8081 or reconfigure their Bridge server with `http.management.port`.
+
+## Rejected alternatives
+
+### Automating certificate management as part of this proposal
+
+- Currently the operator does not manage certificates for other custom resources, as they are designed to be independent, it only manages certificates for its internal components such as Cruise Control and Kafka Exporter, that are deployed as part of the Kafka custom resource. This will introduce a new behaviour to the operator and would require taking other custom resources such as KafkaConnect into consideration as well, because we would potentially need to manage certificates for it too. We are currently in the process of refactoring how the operator manages certificates to support external certificate manager as proposed [here](https://github.com/strimzi/proposals/pull/135), therefore it is not the right time to introduce such new behaviour, as it would not be future proof. Instead, it was decided to leave it for future improvement. 
+
+- Continue using a single interface and only use separate listeners when SSL is enabled so that we don't introduce a breaking change to existing users. Given that we will be introducing the first major version of Bridge, introducing a breaking change is acceptable and it makes much more sense to have separate listeners for better separation of concerns regardless of TLS. In the future, we are likely to implement authentication and authorization for HTTP Bridge. By having separate listeners, th admin endpoints will not be affected and implementation will be simpler. 
```

Comment:
```suggestion
- Continue using a single interface and only use separate listeners when SSL is enabled so that we don't introduce a breaking change to existing users. Given that we will be introducing the first major version of Bridge, introducing a breaking change is acceptable and it makes much more sense to have separate listeners for better separation of concerns, , whether or not TLS is enabled. In the future, we are likely to implement authentication and authorization for HTTP Bridge. By having separate listeners, the admin endpoints will not be affected and implementation will be simpler. 
```